packages/client/src/components/ViewPageButton/ViewPageButton.component.js

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './ViewPageButton.styles.css';
 
-export const ViewPageButton = ({ label, backgroundColor }) => {
+export const ViewPageButton = ({ label, backgroundColor, onClick }) => {
```

Comment:
I'm not sure I understand why this is needed?

Reply:
We can send a onClick handler function.